test/celeritas/phys/ProcessBuilder.test.cc

```diff
@@ -446,6 +447,54 @@ TEST_F(ProcessBuilderTest, rayleigh)
     }
 }
 
+TEST_F(ProcessBuilderTest, coulomb)
+{
+    Options pbopts;
+    pbopts.brem_combined = false;
+    ProcessBuilder build_process(
+        this->import_data(), this->particle(), this->material(), pbopts);
+
+    // Create process
+    auto process = build_process(IPC::coulomb_scat);
+    EXPECT_PROCESS_TYPE(CoulombScatteringProcess, process.get());
+
+    // Test model
+    auto models = process->build_models(ActionIdIter{});
+    ASSERT_EQ(1, models.size());
+    ASSERT_TRUE(models.front());
+    EXPECT_EQ("coulomb-wentzel", models.front()->label());
+
+    // Applicabilities for electron and positron
+    auto all_applic = models.front()->applicability();
+    ASSERT_EQ(2, all_applic.size());
+    Applicability applic = *all_applic.begin();
+
+    for (auto mat_id : range(MaterialId{this->material()->num_materials()}))
+    {
+        // Test step limits
+        {
+            applic.material = mat_id;
+            auto builders = process->step_limits(applic);
+            EXPECT_TRUE(builders[VGT::macro_xs]);
+            EXPECT_FALSE(builders[VGT::energy_loss]);
+            EXPECT_FALSE(builders[VGT::range]);
+        }
+
+        // Test micro xs
+        for (auto const& model : models)
+        {
+            auto builders = model->micro_xs(applic);
+            auto material = this->material()->get(mat_id);
+            EXPECT_EQ(material.num_elements(), builders.size());
+            for (auto elcomp_idx : range(material.num_elements()))
+            {
+                EXPECT_TRUE(builders[elcomp_idx]);
+            }
+        }
+    }
+}
+
 //---------------------------------------------------------------------------//
 }  // namespace test
 }  // namespace celeritas
+ 
```

Comment:
What editor are you using? 😂

Reply:
the cluster I work on has incompatible versions of clang and cuda modules, so I regularly accidentally make commits without having clang-format installed. I wrote a script that de-loads cuda, loads clang-format, appends a newline to the end of each file in the most recent commit, then adds them and does a `commit --amend` to force re-run the formatting. I didn't think I was having issues with the last PR but I guess now I need to do something less hacky haha

Reply:
Actually it only adds a space at the end of the `} // namespace celeritas` which clang-format normally gets, so I'm not sure where the newlines are coming from

Reply:
If you're doing `echo " " >> blah.hh` that would explain the extra newline. Also if your clang-format module is incompatible, you could try making a `function git-clang-format() { exec /full/path/to/gcf }` and setting the git `clangformat.binary` config option to the full path to clang-format?

Reply:
Even though I have the path to the binary, the shared object libraries it depends on are incompatible with the other loaded modules and get force de-loaded unfortunately. I have modified the script to use the same `git-clang-format HEAD^` that the git post-commit hook uses so I shouldn't have to do any modifying file hacks anymore.